lib/public/views/Runs/ActiveColumns/runDetectorsAsyncQcActiveColumns.js

```diff
@@ -71,7 +76,14 @@ const runDetectorAsyncQualityDisplay = ({ dataPassId, simulationPassId }, runNum
         }
     }
 
-    return qcFlagOverviewPanelLink(qcSummaryDisplay, { dataPassId, simulationPassId, runNumber, dplDetectorId });
+    qcSummaryDisplay = runQualityWasChangedToBad
+        ? tooltip(qcSummaryDisplay, BAD_RUN_IN_ASYNC_QC_MESSAGE)
```

Comment:
Are you sure this message here make sense? To me, it brings absolutely no useful information

Reply:
yes, because it might happen that flags are assigned for all detectors, then there is no X icon, and only red colour of a row remain to signalise that run is bad, what's not sufficient